k-distribution/tests/regression-new/checkWarns/checkUnusedSymbol.k.out

```diff
@@ -1,5 +1,5 @@
 [Error] Compiler: Had 1 structural errors.
 [Error] Compiler: Symbol 'foo(_)_CHECKUNUSEDSYMBOL_Foo_Int' defined but not
-used.
+used. Add the 'unused' attribute if this is intentional.
 	Source(checkUnusedSymbol.k)
```

Comment:
Add a test where we _do_ have the `unused` attribute on a klabel that is _not_ a cell klabel, and it is unused (and there should be no warning).